src/containers/find-course/courses-filter-block/CoursesFilterBlock.constants.ts

```diff
@@ -1,4 +1,4 @@
 export const sortTranslationKeys = [
-  { title: 'myCoursesPage.sortTitles.newest', value: 'createdAt' },
-  { title: 'myCoursesPage.sortTitles.oldest', value: 'createdAt' }
+  { title: 'myCoursesPage.sortTitles.newest', value: 'asc' },
```

Comment:
Values like asc desc etc, should be moved to variables or enums values.

Reply:
done